scripts/transformer_trainer.py

```diff
@@ -0,0 +1,60 @@
+import transformers
+import datasets
+import sklearn.metrics
+import functools
+import numpy as np
+import os
+
+def get_model(params, db_config_base, model_nm):
+    db_config = db_config_base
+    if params is not None:
+        db_config.update({"cls_dropout": params["cls_dropout"]})
+    db_config.update({"num_labels": 2})
+    model = transformers.AutoModelForSequenceClassification.from_pretrained(
+        model_nm, config=db_config
+    )
+    return model
+
+def compute_metrics(eval_pred):
+    predictions, labels = eval_pred
+    predictions = predictions.argmax(axis=-1)
+    classification_report = sklearn.metrics.classification_report(
+        labels, predictions, target_names=["not raining", "raining"], output_dict=True
+    )
+    f1_not_raining = classification_report["not raining"]["f1-score"]
+    f1_raining = classification_report["raining"]["f1-score"]
+    return {"f1_not_raining": f1_not_raining, "f1_raining": f1_raining}
+def get_trainer(dataset, db_config_base, model_nm, FOLDER_TO_OUTPUT, parameters, tokenizer):
```

Comment:
```suggestion
def get_trainer(dataset, db_config_base, model_nm, folder_to_output, parameters, tokenizer):
```
variable names should be lower case